### DISCUSSION TITLE
Cannot initate osm2pgsql.mark_way with Lua flex import

### DISCUSSION THREAD

**Discussion by Weltstaat** [osm2pgsql-dev/osm2pgsql#1573] [Q&A] [answered]
Hi All,

I'm trying to get relation tags into a table of osm highways using the Lua flex import. I mainly relied on this example: https://www.grulic.org.ar/~mdione/glob/posts/bus-routes-with-osm2pgsql-flex-backend/

However the script fails when calling osm2pgsql.mark_way

I get the error message: attempt to call a nil value (field 'mark_way')

I have the suspicion it has something to do with my osm2pgsql setup.

Any ideas on this?

Thanks!

**Comment by joto** [comment, marked as answer]
That script is using an old version of osm2pgsql. `mark_way` is not available any more. See https://github.com/openstreetmap/osm2pgsql/blob/master/flex-config/route-relations.lua for a working example how this is done these days.
